release-notes/opensearch.anomaly-detection.release-notes-1.0.0.0-beta1.md

```diff
@@ -0,0 +1,24 @@
+## Version 1.0.0.0-beta1 Release Notes
+
+Compatible with OpenSearch 1.0.0
+
+### Infrastructure
+
+* Fix: use AWS_SECRET_ACCESS_KEY. ([#3](https://github.com/opensearch-project/anomaly-detection/pull/3))
+* Delete ad-cli ([#5](https://github.com/opensearch-project/anomaly-detection/pull/5))
+* Fix access key typo, added VSCode gitignores ([#8](https://github.com/opensearch-project/anomaly-detection/pull/8))
```

Comment:
Same, all the 3 below are not useful in release notes, just things we introduced then fixed.

Reply:
Do we have any guideline for what should be included in release notes in OpenSearch release? I remember we include all commits in ODFE release.